packages/config/src/env/main.js

```diff
@@ -78,11 +79,17 @@ const getGeneralEnv = async function ({
   context,
 }) {
   const gitEnv = await getGitEnv(buildDir, branch)
+
+  const DEPLOY_ID = deployId || 0
+  const BUILD_ID = buildId || 0
+
   return removeFalsy({
     SITE_ID: id,
     SITE_NAME: name,
-    DEPLOY_ID: deployId,
-    BUILD_ID: buildId,
+    DEPLOY_ID,
+    BUILD_ID,
+    DEPLOY_URL: `https://${DEPLOY_ID}--${name}.netlify.app`,
+    DEPLOY_PRIME_URL: `https://${branch}-${BUILD_ID}--${name}.netlify.app`,
```

Comment:
```suggestion
    DEPLOY_PRIME_URL: `https://${branch}--${name}.netlify.app`,
```

The `BUILD_ID` is not part of `DEPLOY_PRIME_URL` in production.